docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -364,18 +364,18 @@ Also, see [Types for compiler options](#types-for-compiler-options).
+| `metaInfo` | This attribute is deprecated and will be removed in the future. It is left only for compatibility with the legacy backend                                                                                                                | | false                            |
+| `moduleKind` | The kind of JS module generated by the compiler                                                                                                                                                                                          | "plain", "amd", "commonjs", "umd". Also, see [Types for compiler options](#types-for-compiler-options) | null                        |
+| `sourceMap` | Generate source map                                                                                                                                                                                                                      | | false                       |
+| `sourceMapEmbedSources` | Embed source files into the source map                                                                                                                                                                                                   | "never", "always", "inlining". Also, see [Types for compiler options](#types-for-compiler-options) | null                            |
+| `sourceMapNamesPolicy` | Add variable and function names that you declared in Kotlin code into the source map. For more information on the behavior, see our [compiler reference](compiler-reference.md#source-map-names-policy-simple-names-fully-qualified-names-no) | "simple-names", "fully-qualified-names", "no". Also, see [Types for compiler options](#types-for-compiler-options)                                                              | null                  |
+| `sourceMapPrefix` | Add the specified prefix to paths in the source map                                                                                                                                                                                      |  | null                            |
+| `target` | Generate JS files for specific ECMA version                                                                                                                                                                                              | "es5", "es2015"  | "es5"                        |
+| `typedArrays` | This attribute is deprecated and will be removed in the future. It is left only for compatibility with the legacy backend                                                                                                                                                                                              | | false                            |
```

Comment:
I would not list it here

Reply:
Do you mean removing the `typedArrays` argument from the list?